packages/workers-playground/src/QuickEditor/HTTPTab/HTTPTab.tsx

```diff
@@ -89,22 +93,33 @@ export function HTTPTab() {
 					setIsLoading(false);
 				}
 			}
-		},
-		[previewHash, setPreviewUrl, previewUrl, method, headers, hasBody, body]
-	);
-	const ensureDevtoolsConnected = useRef(false);
```

Comment:
Why has this been deleted?

Reply:
The variable `ensureDevtoolsConnected` was used to send the initial request. This is no longer needed as we have the `useEffect` to keep track of all the config and send a new request when needed.